### ISSUE TITLE
needs IE11 support

### ISSUE THREAD

**Issue by aberkow** [aberkow/a11y-menu#27]

**Comment by briankelleher** [comment]
icons.forEach - nodeList forEach is not supported in IE, however Array is, so just make sure we are calling forEach on an array.  ES6 makes available an Array.from() method that can probably take care of this (which would be properly polyfilled/transpiled).

**Comment by briankelleher** [comment]
https://stackoverflow.com/questions/41142932/ie-supports-foreach-when-invoked-fromthe-console-but-not-when-called-from-t

**Comment by aberkow** [comment]
this should be resolved here (f83246d2057136f8e04219e1ef6f02dff47ba081) and will be released soon. 

I added IE support via `@babel/polyfill` and importing it directly into the `Navigation` class.

**Comment by aberkow** [comment]
yeah... this change breaks the menu if you're not using webpack. back to the drawing board for a better implementation.

**Comment by briankelleher** [comment]
Since you are already providing a dist/ version of Navigation.js, can you just add this to your webpack script directly so it gets compiled at an ie10/11 target?

```
const path = require('path');
const webpack = require('webpack');

module.exports = {
    entry: ['@babel/polyfill', './src/js/index.js'],
...
```

This way you leave the compile target for src/ up to the user, and you still make the Navigation element available as an external.

**Comment by aberkow** [comment]
I tried that, but then I was thinking that it might be good to not require people to use webpack etc. maybe I'm wrong?

I added this to the class to create arrays from nodelists. that way people could just drop the script into their project without needing any build steps. what do you think?

```
createArray(nodeList) {
    return Array.prototype.slice.call(nodeList);
}
```

**Comment by briankelleher** [comment]
I'm confused, how does that require people to use webpack?

The library makes available the src/ and dist/ scripts.  The dist version can be an already compiled version with polyfills included, while the src version can leave the option to the user for how they want to compile their code.  The library uses 'classes' which aren't available in older browsers anyways.

The purpose of changing the particular line about the nodelist was because the code was assuming it was an iterable, which in some browsers it is, and others it isn't.  Subsequently, no polyfill can be created or properly identified because it was technically a valid API.  Changing that code to no longer assume it would be an iterable just made the code more identifiable for babel/polyfill.

**Comment by aberkow** [comment]
ok. this is resolved here f75c43963eb37163e63d50f4977cd06e6b4b94e6.
